packages/demo/frontend/src/App.tsx

```diff
@@ -1,16 +1,26 @@
 import { BrowserRouter as Router, Routes, Route } from 'react-router-dom'
-import Terminal from './components/Terminal'
-import Home from './components/home/Home'
-import { PrivyProvider } from './providers/PrivyProvider'
-import { EarnPage } from './pages/EarnPage'
-import { ROUTES } from './constants/routes'
+import { useEffect } from 'react'
+import Terminal from '@/components/Terminal'
+import Home from '@/components/home/Home'
+import { PrivyProvider } from '@/providers/PrivyProvider'
+import { EarnPage } from '@/pages/EarnPage'
+import Docs from '@/pages/Docs'
+import { ROUTES } from '@/constants/routes'
 
 function App() {
+  // Prevent default scroll restoration to allow manual hash navigation
+  useEffect(() => {
+    if ('scrollRestoration' in window.history) {
+      window.history.scrollRestoration = 'manual'
+    }
+  }, [])
```

Comment:
is this something we want to do application wide? will this potentially have unintended side effects in other parts of the demo app? I dont have a ton of context on what `window.history.scrollRestoration = 'manual'` does, so maybe it's not a big deal.